src/layout/component/discussion-callout.js

```diff
@@ -13,15 +13,17 @@ const CalloutWithButton = ({ btntext, btnurl, btntype, _body, _parents, _ID }) =
 	const module = GetModule( _parents, _ID );
```

Comment:
^do we still need this line?

Reply:
nope, we don't need it anymore, that's removed.